links.opam

```diff
@@ -7,7 +7,7 @@ bug-reports: "https://github.com/links-lang/links/issues"
 license: "GPL-2"
 
 available: [
-  ocaml-version >= "4.04.0" & ocaml-version <= "4.06.0"
+  ocaml-version >= "4.04.0" & ocaml-version <= "4.06.1"
```

Comment:
We should be able to remove the upper bound now, i.e. I think it is save to have `ocaml-version >= "4.04.0`" now.

Reply:
done!